RELEASING.adoc

```diff
@@ -1,80 +1,210 @@
 = Releasing Kiali
 
-This document gives the steps required to perform a release of Kiali. This document is only useful to Kiali release managers.
+This document covers the specifics of releasing Kiali, and is meant as a guide
+for Kiali release managers.
+
+The summary of the process is the following:
+
+* Create a new JIRA issue for the release
```

Comment:
JIRA issue of type...?

Reply:
What do you think it should be? There's just "Dev Task" and "Task" that apply here. Maybe "Release Task" should be a type on JIRA?

Reply:
Task would work if any.